src/components/popover/popover.tsx

```diff
@@ -23,9 +23,10 @@ import { ESCAPE } from '../../util/keycodes';
  * clicking a button or link on the popover itself.
  *
  * :::warning
- * The component is emitting a close event when you click outside its container.
- * However, it’s up to you as consumer to react properly on this event and
- * hide the component (preferably by setting the property `open` to `false`).
+ * The component is emitting a close event when you click outside its container
+ * or press the <kbd>Esc</kbd> key. However, it’s up to you as consumer to react
```

Comment:
@Kiarokh what do you think about adding some styling to the `kbd` element in Kompendium? I think it would be great to better highlight keyboard input :slightly_smiling_face:

Reply:
Looks great. I'll do ;)